spec/dummy/config/locales/manual_address_validator/en.yml

```diff
@@ -17,3 +17,18 @@ en:
               too_long: "The town or city must be no longer than 30 characters"
             postcode:
               too_long: "The postcode must be no longer than 8 characters"
+        test/legacy_manual_address_validatable:
```

Comment:
Ditto of https://github.com/DEFRA/waste-exemptions-engine/pull/341/files#r320812535